examples/gin-compat/adaptor_test.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestFuegoGin(t *testing.T) {
+	e, _ := server()
+
+	t.Run("simply test gin", func(t *testing.T) {
+		r := httptest.NewRequest("GET", "/gin", nil)
+		w := httptest.NewRecorder()
+
+		e.ServeHTTP(w, r)
+
+		require.Equal(t, 200, w.Code)
```

Comment:
```suggestion
		require.Equal(t, http.StatusOK, w.Code)
```

Reply:
I find `http.StatusOK` less readable than 200. HTTP status code are common and accepted. And Go does not have enums anyway, so it does not have great value!